src/main/java/spim/fiji/datasetmanager/LightSheetZ1.java

```diff
@@ -1,11 +1,13 @@
 package spim.fiji.datasetmanager;
 
 import fiji.util.gui.GenericDialogPlus;
+import ij.IJ;
 import ij.gui.GenericDialog;
 
-import java.awt.Font;
+import java.awt.*;
```

Comment:
Please do not do these import * things ... I like to know what exactly is required.